extensions/hetzner/src/searchServer.tsx

```diff
@@ -0,0 +1,199 @@
+import { ActionPanel, List, Action, Color, Icon } from "@raycast/api";
+import React, { useEffect, useState } from "react";
+import {
+  getAllServers,
+  rebootServer,
+  shutdownServer,
+  startServer,
+  stopServer,
+} from "./actions/server-api";
+import { HetznerServer, HetznerStatus } from "./models/server";
+import { getConfig } from "./config";
+
+export default function Command() {
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```

Otherwise you'll see [empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states)